docs/sites.yml

```
@@ -5124,3 +5124,15 @@
   built_by: Pattyrn
   built_by_url: https://twitter.com/Pattyrn4
   featured: false
+- title: Intranet Italia Day
+  main_url: https://www.ariadnedigital.it/en
+  url: https://www.intranetitaliaday.it/en
+  description: >
+    The Italian event dedicated to the digital workplace that focuses on planning, governance and company intranet management
+  categories:
+    - Event
+    - Intranet
+    - Conference
+  built_by: Ariadne Digital
+  built_by_url: https://www.ariadnedigital.it
+  featured: false 
```

```suggestion
  featured: false
```
This might fix linting error.